packages/react-bootstrap-table2/src/row.js

```diff
@@ -16,13 +16,11 @@ const Row = (props) => {
     editable: editableRow
   } = props;
   const {
-    ridx: editingRowIdx,
-    cidx: editingColIdx,
     mode,
     onStart,
-    onEscape,
-    onComplete,
-    blurToSave
+    ridx: editingRowIdx,
+    cidx: editingColIdx,
+    ...rest
   } = cellEdit;
```

Comment:
Hi @AllenFang,

Sorry for comment late. I saw `cellEdit` was passed as prop but neither marked as `required` nor with `defaultProps`. Therefore,  I have one suggestion to set defaultProp of `cellEdit` to empty object or anthing else to prevent missing prop issue :)